sankeys/serializers.py

```
@@ -0,0 +1,31 @@
+    class Meta:
+        model = SankeyData
+        fields = ['company_name', 'fiscal_year', 'total_revenue', 'is_loss', 'segments', 'expenses']
+
+    def get_total_revenue(self, obj):
+        return obj.raw_values.get('total_revenue', 0)
+
+    def get_segments(self, obj):
+        """왼쪽 노드: [ {name: '반도체', value: 100}, ... ]"""
+        # 중앙 매출액 노드로 들어오는 모든 source 값만 추출
+        target_name = obj.nodes[0]['name']
+        return [{"name": l['source'], "value": l['value']} for l in obj.links if l['target'] == target_name]
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)